src/Moryx.ControlSystem/Cells/Session.cs

```diff
@@ -125,6 +125,20 @@ public static ReadyToWork StartSession(ActivityClassification classification, Re
             return CreateSession(classification, type, ProcessReference.InstanceIdentity(identity), constraints);
         }
 
+
+        /// <summary>
+        /// Creates a new <see cref="Session"/> for the <paramref name="unknown"/> activity
+        /// with a new session context and marks the activity as failed.
+        /// </summary>
+        /// <param name="unknown"></param>
+        public static UnknownActivityAborted WrapUnknownActivity(IActivity unknown)
+        {
+            unknown.Fail();
```

Comment:
Why is the unknown activity failed here and also in the UnknownActivityAborted constructor?